samples/resources/README.md

```diff
@@ -4,28 +4,29 @@ languages:
 - python
 products:
 - azure
-description: "These samples will show you how to manage reosurce in resource sdk."
+description: "These samples will show you how to manage Resources in resource sdk."
 urlFragment: azure-samples-python-management
 ---
 
 # Python Management SDK Samples
 
-These samples will show you how to manage reosurces in resource sdk.
+These samples will show you how to manage Resources in resource sdk.
```

Comment:
could you change this sentence to

These samples will show you how to show to manage Resources using Azure Python SDK